### DISCUSSION TITLE
Screen Capture not working Correctly

### DISCUSSION THREAD

**Discussion by FMGENT** [apohl79/audiogridder#980] [Help]
Running Mac Pro cheese grater with newly build AMD PC as a server. The server connects easily but when selected the mixing plugin is not displayed/opened properly. It is minimized to the top left of the display area of the audiogridder plugin. I am able to open the selected mixing plugin using a different display monitor and it opens but control of the mixing plugin is lost back on the audiogridder plugin. Any help with this would be appreciated. Thanks.

**Comment by apohl79** [comment]
Do you have scaling enabled on windows? If so, can you try setting it to 100% and see if that helps?

**Comment by FMGENT** [comment]
Hi Andreas,

Thanks for getting back to me. Scaling is set to 100%, but doesn’t make a
difference. Tried sending some pics but too large for email. I tried the
legacy mode screen capture but I have no control of the captured image on
the mix screen.

Thanks for your help with this.

Rawle.

**Reply by apohl79** [reply]
Is this happening with any plugin?

**Comment by FMGENT** [comment]
Hi, yes. It is happening with any plugin.

My mixing computer is a Mac and the server is on windows, if that means
anything.

Regards,

Rawle.

**Reply by apohl79** [reply]
Do you have a screen connected to your windows server?

**Comment by FMGENT** [comment]
It’s on a KVM switch, so I can go back and forth from the main computer to
the server.

Rawle.

**Reply by apohl79** [reply]
Ok, that's probably the issue. Once you disconnect the display from windows (when you switch to the other computer), screen capturing stops working.

Try running VNC. Install a server on windows, and open a VNC session from your mac (macos has a VNC client builtin). Screen capturing should continue to work now.

**Reply by rncwalker** [reply]
FYI
I use a KVM switch and have no problems with the capturing not working. If you use RDP and disconnect the connection, then you will loose capturing.

Robert :)

**Comment by FMGENT** [comment]
Server

**Comment by FMGENT** [comment]
I thought about that, but even when I don’t switch to the other (server)
computer the image does not display and from what is visible it does not
have the plugin GUI opened. Thoughts?

**Reply by apohl79** [reply]
Please provide the logfiles of the server and plugin. https://audiogridder.com/bug-report-diagnostics-locations/

**Reply by apohl79** [reply]
btw, did you try vnc? and does screen capturing for vnc work?

**Comment by jacomart** [comment]
Got the same issue! Server and plugin ver. 1.1.1 on Windows 10, client on MacOs Monterey 12.6, DAW Logic Pro 10.7.4.
VNC and Teamviewer works correctly, but the plugin GUI on Logic shows only a bunch of controls as the attached image (every plugin i load do the same). On another (older) Mac with High Sierra (10.13) and Logic Pro 10.4 everythig goes right. I'm a little confused, where I'm wrong? 
<img width="696" alt="Schermata 2022-10-20 alle 14 14 35" src="https://user-images.githubusercontent.com/116279253/196952285-aa4963f3-e8de-4826-bb7e-becff9e2ee22.png">

**Reply by rncwalker** [reply]
Try this (Play the video)

![Generic Editor](https://user-images.githubusercontent.com/112222499/198917003-0a22a2a3-9455-4ca2-9495-0682f166e3f4.gif)



Robert
